### ISSUE TITLE
pygame?

### ISSUE THREAD

**Issue by fragmentedaven** [CheranMahalingam/Snake_Hamiltonian_Solver#2]
The whole piece will not run because it lacks a module named pygame. 

Here is the error message 

ModuleNotFoundError: No module named 'pygame'

**Comment by CheranMahalingam** [comment]
Sorry, this project is old, I should have provided instructions for setup.
It uses python's pipenv to manage dependencies so if you don't have it installed run,
`pip install pipenv`
then to install the dependencies, while in this repository's folder run,
`pipenv install`
and finally to activate the environment run,
`pipenv shell`
Now when you run `python Snake_Solver.py` it should not have this error.
I can confirm that this worked on Windows, let me know if this solves your issue.

**Comment by fragmentedaven** [comment]
Thank you so much. I just figured that out. I am actually at a roadblock trying to get PATH to work.  https://www.pythontutorial.net/python-basics/install-pipenv-windows/I was following along with this until I got to the part where I need to “Second, replace your <username> in the following paths and add them to the PATH environment variable:” And, now I need to figure out how to actually do that because this does not tell me how. I am also running into this issue on other projects as well. If you know how to do this, any help you have would be greatly appreciated.  Also, thank you for being so quick to get back to me with so little information.   Sent from Mail for Windows From: CheranMahalingamSent: Friday, May 20, 2022 7:35 PMTo: CheranMahalingam/Snake_Hamiltonian_Cycle_SolverCc: fragmentedaven; AuthorSubject: Re: [CheranMahalingam/Snake_Hamiltonian_Cycle_Solver] pygame? (Issue #2) Sorry, this project is old, I should have provided instructions for setup.It uses python's pipenv to manage dependencies so if you don't have it installed run,pip install pipenvthen to install the dependencies, while in this repository's folder run,pipenv installand finally to activate the environment run,pipenv shellNow when you run python Snake_Solver.py it should not have this error.I can confirm that this worked on Windows, let me know if this solves your issue.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
